packages/quantic/force-app/main/default/lwc/quanticResultChildren/quanticResultChildren.js

```diff
@@ -182,6 +182,10 @@ export default class QuanticResultChildren extends LightningElement {
     return !this.areMoreResultsAvailable;
   }
 
+  get hasCollectionChildren() {
```

Comment:
As mentioned above, we can update the logic that controls the display of the noChildrenMessage in the already existing condition: `shouldDisplayNoChildrenMessage`

Reply:
Yea now since you are in this same component, you could just add a new condition to the `shouldDisplayNoChildrenMessage` however make sure no unnecessary HTML will be renderer if the component doesn't have any children.